tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -363,7 +363,7 @@
                 required=False,
                 validators=[
                     category3.orValidators([
-                        category3.isBetween(0, 3, inclusive=True),
+                        category3.isBetween(1, 3, inclusive=True),
```

Comment:
cant test this easily in deployed env because field is not required. but code changes LGTM ✔️